auto_nag/scripts/severity_high_security.py

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+HIGH_SECURITY_KEYWORDS = ["sec-high", "sec-critical"]
+
+
+class SeverityHighSecurity(BzCleaner):
+    def description(self):
+        return "Bugs with high security keywords whose set to low severity"
+
+    def has_needinfo(self):
+        return True
+
+    def handle_bug(self, bug, data):
+        security_keywords = [
+            keyword for keyword in bug["keywords"] if keyword in HIGH_SECURITY_KEYWORDS
+        ]
+        assert len(security_keywords) == 1
+        security_keyword = security_keywords[0]
+
+        bugid = str(bug["id"])
+        data[bugid] = {
+            "security_keyword": security_keyword,
+            "severity": bug["severity"],
+        }
+
+        return bug
+
+    def get_extra_for_needinfo_template(self):
+        return self.get_data()
```

Comment:
This is going to return an empty dict, which will make `extra[bugid]["severity"]` in the needinfo template fail (as bugid is not in extra, given that extra is an empty dict).

In this script you don't actually need to implement this `get_extra_for_needinfo_template`, and you can simply replace `extra[bugid]["severity"]` with `severity` (and similarly for `extra[bugid]["security_keyword"]`.

Reply:
In the template rendering, only `bugid` and `extra` are passed:

https://github.com/suhaibmujahid/relman-auto-nag/blob/d23b911d8aa428dde533e7e6e38c9182e21896b0/auto_nag/bzcleaner.py#L440-L446

I will use the same workaround that I used in `severity_inconsistency.py`:

https://github.com/suhaibmujahid/relman-auto-nag/blob/08f0ce1d2193345706cff8ba253d01fe05929ed0/auto_nag/scripts/severity_inconsistency.py#L30-L35

Maybe a future refactoring should address this :)

Reply:
Fixed in 7836438dc33ab217865b339d8f1f54295463b98d

Reply:
What I suggested was actually working for me (I tested it locally)

Reply:
Done in f65679d37c7128f47ff4db0389fc796ee5fbc5fa